archunit-junit/junit5/engine/build.gradle

```diff
@@ -62,7 +68,3 @@ def configureDependencies = { deps ->
     }
 }
 this.with project(':archunit-junit').configureJUnitArchive(configureDependencies)
-
-singlePackageExport {
```

Comment:
Not sure what to do about this one (should I just move everything from `internal/filtering` to `internal`? If the other PRs are merged there will be a lot more filtering stuff)